about.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
looks like HTML validators don't like relative paths to have spaces in it! In the future, make sure your pictures, files, what have you use "-" or "_" to represent spaces!